src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -277,15 +281,17 @@ function WeeklySummaryMessage({ summary, weekIndex }) {
   );
 }
 
-function TeamCodeRow({ canEditTeamCode, summary }) {
+function TeamCodeRow({ canEditTeamCode, handleTeamCodeChange, summary }) {
   const [teamCode, setTeamCode] = useState(summary.teamCode);
+  const [_teamCode, set_TeamCode] = useState(teamCode);
```

Comment:
the state name "_teamCode" runs parallel to original "teamCode" and could be very confusing